dotnet/src/Functions/Functions.OpenApi/Serialization/FormStyleParameterSerializer.cs

```diff
@@ -24,7 +24,8 @@ public static string Serialize(RestApiOperationParameter parameter, JsonNode arg
         Verify.NotNull(parameter);
         Verify.NotNull(argument);
 
-        if (parameter.Style != RestApiOperationParameterStyle.Form)
+        var style = parameter.Style ?? RestApiOperationParameterStyle.Form;
+        if (style != RestApiOperationParameterStyle.Form)
         {
             throw new NotSupportedException($"Unsupported Rest API operation parameter style '{parameter.Style}' for parameter '{parameter.Name}'");
```

Comment:
Should we add more information about what parameter style is supported?
```suggestion
            throw new NotSupportedException($"Unsupported Rest API operation parameter style '{parameter.Style}' for parameter '{parameter.Name}'. Only '${nameof(RestApiOperationParameterStyle.Form)}' parameter style is supported.");
```

Reply:
This change will be needed in multiple places, so will leave it out of this PR